### ISSUE TITLE
Ticket Work Flow Suggestion

### ISSUE THREAD

**Issue by mbiolsi** [ladybirdweb/faveo-helpdesk#119]
A few recommendations for the way tickets work in Faveo.
First, there are many times that an agent has to create the ticket for the client. Currently, we can do that, but the ticket shows "Open" after making the ticket. Sometimes, we use tickets to request information from our clients for their projects. If there was an option to set the status of the ticket when creating it (as you add the custom status) that would be great. Then we could create a ticket and set the status to "Pending Client" all at once. Presently if we create a ticket for our client, it is in the OPEN section which in my mind means we have to act on it and the only way to change it is to answer it which looks weird.
Second, when a client replies, it is NOT obvious that we need to take action. Can we have tickets replied to be clients go back to OPEN? In my mind, OPEN means we have to do something, ANSWERED means we have answered them and the ball is in their court.  
ICONS: what about icons, like we are used to in all of our email apps, that show if the ticket has been replied to? So, if we have replied, then there is an arrow or something to indicate that we have answered and are waiting for the client? It is very challenging to know which tickets we have to respond to and which we are waiting on them form. I know they are bold, but again, if WE created the ticket, the it is bold and we are not waiting for them. 

On a side note: I understand that we might be using the helpdesk in ways that others are not, so these might not fit enough users to justify considering.

**Comment by sujit223** [comment]
Sure we will try to look into it.

**Comment by bhanu2217** [comment]
Also have a look here https://github.com/ladybirdweb/faveo-helpdesk/wiki/Faveo-Feedback-&-Customisation

**Comment by bhanu2217** [comment]
Custom Ticket status is part of our roadmap, and you should hopefully see them in August.

**Regarding Differentiate between the tickets where last reply is client or agent:**
- When the last reply is from client the ticket is highlighted, which is more like unread emails
- When the last reply is agent ticket is not highlighted, more like unread emails.

**Current work around to your problem:**
- Create ticket on behalf of client
- After creating ticket, reply as agent and ticket will be unread

**Solution:**
- We can have a button in the system which says **_Mark as read_** and this will change again once the client replies
- We can have a new ticket source as **_Agent_**, If the ticket source is agent than new ticket will be marked as unread until client replies
